### ISSUE TITLE
What is the memory of V 100

### ISSUE THREAD

**Issue by onlyonewater** [jonmun/MM-SADA-code#5]
Hi, in your paper, you say you use 8 V100 to train the model, so the V100 has 16GB or 32GB memory?? And you set the batch size as 128, it is too large. If the batch size becomes smaller, will the resulting decrease??

**Comment by avijit9** [comment]
Hi,

I have been training these models with batch size 64 on 4 1080Ti Gpus. The performances seem to be closer to what reported.

**Comment by onlyonewater** [comment]
Hi, @avijit9 , thank you for your reply, If it's convenient, can you tell me your experimental hyper-parameters? For example: learning rate, etc.

**Comment by avijit9** [comment]
I kept everything exactly the same as given in this repo except the batch size. 

However, if you are having trouble finding the exact performance, you can scale the learning rate with a factor by which the batch size is increased/decreased. For example, if your batch size is 64 instead of 128, you should multiply the original learning rate with a factor = (64/128). 

But the exact hyperparameters here works.

**Comment by onlyonewater** [comment]
oh, very thanks, I get it!!
